packages/react/src/components/Popup/Popup.tsx

```diff
@@ -474,6 +476,8 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
   }
 
   private trySetOpen(newValue: boolean, eventArgs: any) {
+    // when new state 'open' === 'true', save the last focused element
+    newValue && this.updateTriggerFocusableDomElement()
```

Comment:
```suggestion
    if (newValue) this.updateTriggerFocusableDomElement()
```

```suggestion
    if (newValue) {
      this.updateTriggerFocusableDomElement()
    }
```

Let's make it more readable.